src/scripts/process_us_ca.py

```
@@ -0,0 +1,198 @@
+                print(f"   Running... {count} (Sample Title: {rows[0]['law_title']})")
+    except Exception as e: print(f"US Error: {e}")
+
+    # 2. 캐나다 데이터 (Multi_Legal_Pile)
+    print("\n🇨🇦 [CA] 수집 중...")
+    try:
+        ca_ds = load_dataset("joelniklaus/Multi_Legal_Pile", "en_legislation", split="train", streaming=True, trust_remote_code=True)
+        ca_ds = ca_ds.shuffle(seed=42, buffer_size=30000)
+        
+        count = 0
+        for item in ca_ds:
+            if count >= TARGET_PER_COUNTRY: break
```

Ruff (0.14.7)

Multiple statements on one line (colon)

(E701)